NgsildAgent/util/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Why is this copyrighted by Intel? Is it taken directly from OISP? I guess, partially it needs also copyright from Microstep or IBSystems